Themes/default/PersonalMessage.template.php

```diff
@@ -463,11 +464,11 @@ function template_single_pm($message)
 				</ul>
 			</div><!-- .poster -->
 			<div class="postarea">
-				<div class="flow_hidden">
-					<div class="keyinfo">
-						<h5 id="subject_', $message['id'], '">
```

Comment:
why the convert of h5 to div?

Reply:
Display template for converted to a div, we want it to be consistent, specially for css selectors

Reply:
Should I add back the h5 back inside so titles are bold? @Sesquipedalian

Reply:
I changed this back in 7f32adc and removed the bold for the postinfo in pms